modules/cron/bin/cron.php

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This script can be used to invoke cron jobs from cli.
+ * You most likely want to execute as the user running your webserver.
+ * Example:  su -s "/bin/sh" -c "php /var/simplesamlphp/modules/cron/bin/cron.php -t hourly" apache
+ */
+
+// This is the base directory of the SimpleSAMLphp installation
+$baseDir = dirname(dirname(dirname(dirname(__FILE__))));
+
+// Add library autoloader.
+require_once($baseDir . '/lib/_autoload.php');
+
+if (!SimpleSAML\Module::isModuleEnabled('cron')) {
+    echo("You need to enable the cron module before this script can be used.\n");
+    echo("You can enable it by running the following command:\n");
+    echo('  echo >"' . $baseDir . '/modules/cron/enable' . "\"\n");
+    exit(1);
+}
+
+$options = getopt("t:");
+if (posix_getuid() === 0) {
```

Comment:
Probably add an `function_exists` there because `posix_getuid` is not always available (requires posix module, not on Windows).

Reply:
thanks for the tip. I've added the check.